### PR TITLE
masha checking aligner

### PR DESCRIPTION
**Pull request by maxaalexeeva**

### CODE REVIEW

File: text_reading/src/main/scala/org/clulab/aske/automates/alignment/Aligner.scala [hunk 1]
```diff
@@ -35,6 +35,25 @@ class VariableEditDistanceAligner {
   }
 }
 
+class MentionEditDistanceAligner {
+  val relevantArgs = Set("definition")
```
Comment (BeckySharp) [inline text_reading/src/main/scala/org/clulab/aske/automates/alignment/Aligner.scala]: please pass into baseclass

File: text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala [hunk 2]
```diff
@@ -121,6 +121,17 @@ class OdinActions(val taxonomy: Taxonomy, expansionHandler: Option[ExpansionHand
     mentions.flatMap(mkDefinitionMention)
   }
 
+  def lookLikeAVariableInAnyRel(mentions: Seq[Mention], state: State): Seq[Mention] = {
```
Comment (BeckySharp) [inline text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala]: Instead, can you modify the base version with something like words = m match {...}

File: text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala [hunk 3]
```diff
@@ -146,8 +146,9 @@ object ExtractAndAlign {
 //    }
 //    pw.close()
     // ----------------------------------
-
-    val commentToTextAlignments = w2vAligner.alignMentions(commentDefinitionMentions, textDefinitionMentions)
+    val defEditAligner = new MentionEditDistanceAligner
```
Comment (BeckySharp) [inline text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala]: if you can specify this in the config file -- bonus points

Comment (BeckySharp) [reply text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala]: @maxaalexeeva  Are you planning on going for the bonus points? Or should I merge?

Comment (maxaalexeeva) [reply text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAlign.scala]: Bonus points. Thanks for checking! I'll mention you here when it's ready to merge.

**Pull request merged**